libs/financial-aid/shared/src/lib/const.ts

```diff
@@ -38,6 +38,7 @@ export const Routes = {
   municipalityProfile: (id: string) => `/sveitarfelog/${id}`,
   municipalitySettings: `/sveitarfelagsstillingar`,
   municipalities: `/sveitarfelog`,
+  supervisors: `/umsjonaradilar`,
```

Comment:
maybe time to make to obj for example 
settings {
  municipalitySettings: `/sveitarfelagsstillingar`,
  municipalities: `/sveitarfelog`,
  supervisors: `/umsjonaradilar`,
}